packages/xsnap/src/object-inspect.js

```diff
@@ -0,0 +1,489 @@
+/* eslint-disable no-empty */
+/* eslint-disable no-nested-ternary,no-use-before-define */
```

Comment:
The justification for `no-use-before-define` is to reduce the diffs from the code we started from?

I'm not getting much benefit. github thinks this is a complete new file. :-/

Reply:
> The justification for `no-use-before-define` is to reduce the diffs from the code we started from?

Not to reduce the diffs, to reduce the porting effort.